code/modules/clothing/head/helmet.dm

```diff
@@ -456,14 +458,23 @@
 	if(policy)
 		to_chat(magnification, policy)
 	icon_state = "[icon_state]up"
+	REMOVE_TRAIT(magnification, TRAIT_PRIMITIVE, SPECIES_TRAIT) //Monkeys with sentience should be able to use less primitive tools.
 
 /obj/item/clothing/head/helmet/monkey_sentience/Destroy()
-	disconnect()
+	if(magnification)
```

Comment:
Shouldn't this be doing all the secondary disconnect effects if monkey still exists?

Reply:
:-1: Destroy should not be performing these kinds of side effects at all.

Reply:
I meant Unregisters and such, though you're right the sound/message stuff should be in destruction instead.